practice_app/apps/post/templates/postindex.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+  	<title>Home</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css">
+    <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+    <style>
+        table, th, td {
+            border: 1px solid;
+        }
+        .center {
+            margin-left: auto;
+            margin-right: auto;
+            justify-content: center;
+        }
+        form {
+            margin: 0 auto;
+            width:250px;
+        }
+    </style>
+</head>
+<body>
+
+
+    <div style="background-color:plum; !important" class="jumbotron text-center bg-primary">
+            <h2> Welcome </h2>
+    </div>
+
+      {% if success %}
+        <div class="alert alert-success" role="alert" >
+        <h4 class="alert-heading">Post is successfully created .</h4>
+
+    </div>
+
+    {% endif %}
+
+    <form id="postform" class="form-group" method="GET" action="{%url 'getPost' %}" >
+        {% csrf_token %}
+  <label for="id">Post ID:</label><br>
+  <input type="text" id="id" name="id" required><br>
+ <button type="submit" style="margin-top: 10px;"  class="btn btn-success">Get Post Info</button>
+</form>
+    <form id="post" class="form-group" method="POST" action="{%url 'createPost' %}" >
+        {% csrf_token %}
+  <label for="id">Create New Post:</label><br>
+        <input type="text"style="margin-top: 1px;" placeholder="title" id="title" name="title" required><br>
+        <input type="text"style="margin-top: 1px;"  placeholder="description" id="description" name="description" required><br>
+        <input type="text"style="margin-top: 1px;"  placeholder="post type label" id="post_type_label" name="post_type_label" required><br>
```

Comment:
Maybe, you can write placeholder's strating with capital letter to a little bit better UI.

Reply:
Thanks, I have updated the html file.